components/brave_rewards/resources/page/reducers/rewards_reducer.ts

```diff
@@ -453,6 +453,8 @@ const rewardsReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State
     }
     case types.ON_COMPLETE_RESET: {
       if (action.payload.success) {
+        onboardingCompletedStore.clear()
+        chrome.send('brave_rewards.getOnboardingStatus')
```

Comment:
Shouldn't we make this change to `components/brave_rewards/resources/android_page/reducers/rewards_reducer.ts` as well?

Reply:
I did have it in there initially, but there's no onboarding overlay on the Android settings page. Maybe I should add it to the reducer anyway for consistency though? WDYT?

Reply:
Yeah, I'd probably still add it.

Reply:
Yep, agreed. I added that in and a couple of missing onboarding-related `case` statements to bring the android page in line with the desktop page.